src/Interface/InterfacesListContainer.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { isEqual, isEmpty, get } from 'lodash';
+import { FieldArray, getFormValues } from 'redux-form';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+
+import InterfacesList from './InterfacesList';
+
+class InterfacesListContainer extends Component {
+  static propTypes = {
+    parentMutator: PropTypes.object,
+    parentResources: PropTypes.object,
+    stripes: PropTypes.shape({
+      store: PropTypes.object
+    }),
+    orgId: PropTypes.string,
+  };
+
+  state = {
+    interfaceArrState: [],
+  };
+
+  static getDerivedStateFromProps(props, { interfaceArrState = [] }) {
```

Comment:
this is pretty long and not readable function. PLease don't copy-paste this part of contacts. It could be done in more elegant and easy to understand way